modules/metastorage-api/src/main/java/org/apache/ignite/internal/metastorage/MetaStorageManager.java

```diff
@@ -81,6 +76,25 @@ public interface MetaStorageManager extends IgniteComponent {
     @Deprecated
     List<Entry> getLocally(byte[] key, long revLowerBound, long revUpperBound);
 
+    /**
+     * Returns an entry by the given key and bounded by the given revision. The entry is obtained
+     * from the local storage.
+     *
+     * @param key The key.
+     * @param revUpperBound The upper bound of revision.
+     * @return Value corresponding to the given key.
+     */
+    Entry getLocally(byte[] key, long revUpperBound);
```

Comment:
A similar method is deprecated and has a TODO, is it applicable here as well?

Reply:
Discussed in a private chat, we agreed to leave the method